test/ecto_adapters_dynamodb_test.exs

```diff
@@ -339,10 +339,15 @@ defmodule Ecto.Adapters.DynamoDB.Test do
               select: p.id)
              |> TestRepo.one() == person.id
 
```

Comment:
Are we testing a scenario where there is only one result pre-filter?

Reply:
Do you mean like

```
from(p in Person,
  where: p.id in ["person-larry", "person-not-exists"]
    and p.first_name == "Joe"
      or p.age >= 72,
  select: p.id)
|> TestRepo.all()
```
?

I'll add a test for that.

Reply:
If "person-not-exists" is an ID for a record that is not in the database, then yes.

Reply:
Cool, thanks - I added a test for that.